### ISSUE TITLE
What should be done about browser focus rings?

### ISSUE THREAD

**Issue by das-sein** [FirefoxUX/photon-components-web#98]
I'm wondering what the opinion is of modifying the default `:-moz-focusring` and `:-webkit-focus-ring-color` behavior. Specifically, to stop them from being displayed. The design spec lists `:focus` styles for elements like radio buttons and check-boxes, but there doesn't seem to be a practice at the moment to mess with the browser focus ring behavior.

I would argue that it interferes with the design language being created by muddling the indicators of various states, but I am interested to know what the professionals on the project feel about this, and if it should be something that is left alone or is changed.

Consider:

![image](https://user-images.githubusercontent.com/6910190/52161934-89c94980-269a-11e9-82f8-08e979a3e095.png)

![image](https://user-images.githubusercontent.com/6910190/52161948-afeee980-269a-11e9-9bac-ba1d252cc5e7.png)
